controller/input_config.json

```diff
@@ -1,8 +1,9 @@
 {
-  "database_type" : "postgres",
```

Comment:
Can you move this file into a directory called "config" and rename it sample_mysql_config.json? It would also be good to add a sample_postgres_config.json file.

Reply:
Should we pass the config file as an argument via command line?

Reply:
Yes and I think it's best to make the path to the config file a required command line argument.

Reply:
Done.